argo-cd.yaml

```diff
@@ -36,14 +36,17 @@ pipeline:
       mkdir -p ${{targets.destdir}}/usr/bin
       mv dist/argocd* ${{targets.destdir}}/usr/bin/
 
-      ln -s /usr/bin/argocd ${{targets.destdir}}/usr/bin/argocd-server
-      ln -s /usr/bin/argocd ${{targets.destdir}}/usr/bin/argocd-repo-server
-      ln -s /usr/bin/argocd ${{targets.destdir}}/usr/bin/argocd-cmp-server
-      ln -s /usr/bin/argocd ${{targets.destdir}}/usr/bin/argocd-application-controller
-      ln -s /usr/bin/argocd ${{targets.destdir}}/usr/bin/argocd-dex
-      ln -s /usr/bin/argocd ${{targets.destdir}}/usr/bin/argocd-notifications
-      ln -s /usr/bin/argocd ${{targets.destdir}}/usr/bin/argocd-applicationset-controller
-      ln -s /usr/bin/argocd ${{targets.destdir}}/usr/bin/argocd-k8s-auth
+      # ArgoCD manifests and helm charts all hardcode the executables path to /usr/local/bin/*
```

Comment:
Would it be possible to add the /usr/local symlinks as a compatibility subpackage?

Reply:
added the symlinks to `/usr/local/bin/*` in an `argo-cd-compat` subpackage